src/main/java/uk/gov/hmcts/ccd/auditlog/AuditInterceptor.java

```diff
@@ -59,24 +61,33 @@ private AuditContext populateHttpSemantics(AuditContext auditContext,
         AuditContext context = (auditContext != null) ? auditContext : new AuditContext();
         context.setHttpStatus(response.getStatus());
 
-        if (request.getMethod() != null && httpMethodList.contains(request.getMethod().toUpperCase())) {
-            context.setHttpMethod(request.getMethod());
+        String requestMethod = request.getMethod();
+        if (requestMethod != null && httpMethodList.contains(requestMethod.toUpperCase())) {
+            context.setHttpMethod(requestMethod);
         } else {
             context.setHttpMethod(BAD_VALUE_TOKEN);
             LOG.error("Error while validating Http Method with value {} as it did not meet validation criteria:{}",
-                request.getMethod(), httpMethodList);
+                encodeString(requestMethod), httpMethodList);
         }
 
         context.setRequestPath(request.getRequestURI());
 
-        if (request.getHeader(REQUEST_ID) == null || request.getHeader(REQUEST_ID).matches(REQUEST_ID_PATTERN)) {
-            context.setRequestId(request.getHeader(REQUEST_ID));
+        String requestId = request.getHeader(REQUEST_ID);
+        if (requestId == null || requestId.matches(REQUEST_ID_PATTERN)) {
+            context.setRequestId(requestId);
         } else {
             context.setRequestId(BAD_VALUE_TOKEN);
             LOG.error("Error while validating Request Id with value {} as it did not meet validation criteria:{}",
```

Comment:
Error while validating Request Id with Base-64 value {} as it did not meet validation criteria:{}